### PR TITLE
Define the partitioning effort in more detail

### PR DESCRIPTION
**Pull request by annevk**

### CODE REVIEW

**Comment by annevk** [comment]
@arturjanc @bakulf @domenic @englehardt @johnwilander @hober @MattMenke2 @shivanigithub @othermaciej @TanviHacks I'd appreciate your thoughts. (And also from everyone else who happens to be passing by.)

**Review by hober — APPROVED** [review]

**Comment by annevk** [comment]
I plan on merging this Monday, unless someone wants a review extension. To be clear, nothing is set in stone at this point.

**Review by englehardt — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -4,10 +4,71 @@ A [Work Item](https://privacycg.github.io/charter.html#work-items)
...
+* Web Authentication
+* WebRTC’s `deviceId` ([w3c/mediacapture-main #675](https://github.com/w3c/mediacapture-main/issues/675))
+* `window.name`
+* Browsing context group's agent cluster map (only observable with popups)
+* Permissions ([Feature Policy](https://w3c.github.io/webappsec-feature-policy/) allows these to be disabled by default when the top-level site is not equal to the current site and require explicit delegation in such cases)
+* Credentials (username and password storage; not a standardized feature)
+
+## Acknowledgments
+
+The author of this document was inspired by Chromium’s NIK, Firefox’s FPI/dFPI/StoragePrincipal, Safari’s ITP, XS-Leaks, and the many people wanting to improve these aspects of the web.
```
Comment (shivanigithub) [inline README.md]: May be mention Chromium's Network Isolation Key (as NIK might not be clear to those who are not familiar with it.)

**Comment by arturjanc** [comment]
LGTM

**Comment by othermaciej** [comment]
This is a good change.

**Comment by annevk** [comment]
Thanks all!

**Pull request merged**